### PR TITLE
Fix notes messages

### PR DESCRIPTION
**Pull request by salbeira**
Fixes Issue #53 by moving initialisation to a dummy plugin object that gets regsitered while keeping the actual plugin code out of the plugin object's scope, removing the circular reference.

### CODE REVIEW

File: resource/decker/support/plugins/decker/ui-anchors.js [hunk 1]
```diff
@@ -19,7 +19,6 @@
 
 class DeckerPlugins {
```
Comment (mbotsch) [inline resource/decker/support/plugins/decker/ui-anchors.js:20]: Please rename to UIAnchorsPlugin or something more meaningful

File: resource/decker/support/plugins/feedback/feedback.js [hunk 2]
```diff
@@ -466,32 +464,42 @@ class Feedback {
   </div>
   <div class="feedback-controls">
     <div class="feedback-controls-wrapper">
-      <span class="votes" title="${text.votes}" aria-label="${text.votes}">${comment.votes > 0 ? comment.votes : ""
-      }</span>
-      <button class="${comment.didvote ? "fas" : "far"} fa-thumbs-up vote ${!isAuthor ? "canvote" : "cantvote"
-      } ${comment.didvote ? "didvote" : ""}"
+      <span class="votes" title="${text.votes}" aria-label="${text.votes}">${
```
Comment (mbotsch) [inline resource/decker/support/plugins/feedback/feedback.js:467]: Why are there so many changes? Is this pure formatting? Formatting changes caused merge conflicts earlier, so let's avoid this by using the same code formater (Prettier)

File: resource/decker/support/plugins/menu/menu.js [hunk 3]
```diff
@@ -669,72 +668,85 @@ class SlideMenu {
...
-    anchors.placeButton(this.open_button, this.position);
-    reveal.addEventListener("slidechanged", () =>
-      this.updateCurrentSlideMark()
-    );
-  }
-}
+      document.body.appendChild(menu.menu.container);
 
-let instance = new SlideMenu("TOP_LEFT");
+      this.getSlideList = () => {
```
Comment (mbotsch) [inline resource/decker/support/plugins/menu/menu.js:729]: what does "this" refer to in this context?

File: resource/decker/support/plugins/decker/ui-anchors.js (hunk 1 above)
Comment (salbeira) [reply resource/decker/support/plugins/decker/ui-anchors.js]: done

**Pull request merged**